File: src/ServiceBundle/Entity/Hebergement.php
<?php

namespace ServiceBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Hebergement
 *
 * @ORM\Table(name="hebergement")
 * @ORM\Entity(repositoryClass="ServiceBundle\Repository\HebergementRepository")
 */
class Hebergement extends Service
{
    /**
     * @ORM\ManyToOne(targetEntity="Service")
     * @ORM\JoinColumn(name="serviceId", referencedColumnName="id")
     */
    private $service;

    public function __construct() {
        parent::__construct();
    }

    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="adresse", type="string", length=255)
     */
    private $adresse;

    /**
     * @var int
     *
     * @ORM\Column(name="nbChambre", type="integer")
     */
    private $nbChambre;

    /**
     * @var int
     *
     * @ORM\Column(name="nbLits", type="integer")
     */
    private $nbLits;

    /**
     * @var int
     *
     * @ORM\Column(name="capacite", type="integer")
     */
    private $capacite;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text")
     */
    private $description;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set adresse
     *
     * @param string $adresse
     *
     * @return Hebergement
     */
    public function setAdresse($adresse)
    {
        $this->adresse = $adresse;

        return $this;
    }

    /**
     * Get adresse
     *
     * @return string
     */
    public function getAdresse()
    {
        return $this->adresse;
    }

    /**
     * Set nbChambre
     *
     * @param integer $nbChambre
     *
     * @return Hebergement
     */
    public function setNbChambre($nbChambre)
    {
        $this->nbChambre = $nbChambre;

        return $this;
    }

    /**
     * Get nbChambre
     *
     * @return int
     */
    public function getNbChambre()
    {
        return $this->nbChambre;
    }

    /**
     * Set nbLits
     *
     * @param integer $nbLits
     *
     * @return Hebergement
     */
    public function setNbLits($nbLits)
    {
        $this->nbLits = $nbLits;

        return $this;
    }

    /**
     * Get nbLits
     *
     * @return int
     */
    public function getNbLits()
    {
        return $this->nbLits;
    }

    /**
     * Set capacite
     *
     * @param integer $capacite
     *
     * @return Hebergement
     */
    public function setCapacite($capacite)
    {
        $this->capacite = $capacite;

        return $this;
    }

    /**
     * Get capacite
     *
     * @return int
     */
    public function getCapacite()
    {
        return $this->capacite;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Hebergement
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }
}

File: web/java/huntkingdom/src/Entities/Hebergement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

/**
 *
 * @author ASUS1
 */
public class Hebergement {
    
    private int id;
    
    private String type;
    
    private float prixParJour;
    
    private  String image;
    
    private String adresse;
    
    private int nbChambre;
    
    private int nbLits;
    
    private int capacite;
    
    private String description;

    public Hebergement() {
    }

    public Hebergement(int id, String type, float prixParJour, String image, String adresse, int nbChambre, int nbLits, int capacite, String description) {
        this.id = id;
        this.type = type;
        this.prixParJour = prixParJour;
        this.image = image;
        this.adresse = adresse;
        this.nbChambre = nbChambre;
        this.nbLits = nbLits;
        this.capacite = capacite;
        this.description = description;
    }

    
    
    
    public Hebergement(String type, float prixParJour, String image, String adresse, int nbChambre, int nbLits, int capacite,String description) {
        this.type = type;
        this.prixParJour = prixParJour;
        this.image = image;
        this.adresse = adresse;
        this.nbChambre = nbChambre;
        this.nbLits = nbLits;
        this.capacite = capacite;
        this.description = description;
    }
    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Hebergement{" + "id=" + id + ", type=" + type + ", prixParJour=" + prixParJour + ", image=" + image + ", adresse=" + adresse + ", nbChambre=" + nbChambre + ", nbLits=" + nbLits + ", capacite=" + capacite + '}';
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public float getPrixParJour() {
        return prixParJour;
    }

    public void setPrixParJour(float prixParJour) {
        this.prixParJour = prixParJour;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public int getNbChambre() {
        return nbChambre;
    }

    public void setNbChambre(int nbChambre) {
        this.nbChambre = nbChambre;
    }

    public int getNbLits() {
        return nbLits;
    }

    public void setNbLits(int nbLits) {
        this.nbLits = nbLits;
    }

    public int getCapacite() {
        return capacite;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setCapacite(int capacite) {
        this.capacite = capacite;
    }
    
    
    
}
